metrics/metrics.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2017, Square, Inc.
+
+// Package metrics is a wrapper of github.com/rcrowley/go-metrics.
+package metrics
+
+import (
+	gometrics "github.com/rcrowley/go-metrics"
+)
+
+type Metrics interface {
+	GetOrRegister(string, interface{})
+}
+
+type metrics struct {
+	registry gometrics.Registry
+}
+
+type Counter interface {
+	Inc(int64)
+	Dec(int64)
+	Count() int64
+	Clear()
+}
+
+type counter struct {
+	counter gometrics.Counter
+}
+
+type Timer interface {
+	Time(func())
+	Percentile(float64) float64
+	Mean() float64
+	Count() int64
```

Comment:
Total time?

Reply:
I don't see what the total time would be useful for (assuming that you mean total time to be the sum of all the times we've recorded). Why do you see it useful to report?

Reply:
Sorry, I meant: "Does this return Total Time?"  i.e. at first, I don't know what a Time.Count would be. I'm playing devil's advocated. Later I saw it's count of events.

Reply:
Ah, I see. Yeah, its's the count of events. There's a comment on the Count() function, so that should be ok?

Reply:
Or actually I can move the code comment from above "Count()" method to inside the interface definition here.